pkgs/by-name/cl/clevis/package.nix

```diff
@@ -72,6 +60,30 @@ stdenv.mkDerivation rec {
     "man"
   ];
 
+  postPatch = ''
+    for f in $(find src/ -type f); do
+      grep -q "/bin/cat" "$f" && substituteInPlace "$f" \
+        --replace-fail '/bin/cat' '${lib.getExe' coreutils "cat"}' || true
+    done
+  '';
```

Comment:
Personally, I think that putting a `postPatch` attribute right after the `patches` attribute is more intuitive, and it also reduces the diff and makes reading the commits of the PR one by one a better experience.